src/server/node.rs

```diff
@@ -218,6 +218,11 @@ where
         self.store.get_id()
     }
 
+    /// Gets the Scheduler of RaftstoreConfigTask, it must be called after start.
+    pub fn get_scheduler(&mut self) -> Scheduler<RefreshConfigTask> {
```

Comment:
`get_scheduler` could be confusing, how about just `refresh_config_scheduler`?

Reply:
Fixed, thanks!